fcrepo-auth-common/src/main/java/org/fcrepo/auth/common/FedoraUserSecurityContext.java

```diff
@@ -39,11 +39,11 @@
     private static final Logger LOGGER = LoggerFactory
             .getLogger(FedoraUserSecurityContext.class);
 
-    private Principal userPrincipal = null;
+    protected Principal userPrincipal = null;
 
-    private FedoraAuthorizationDelegate fad = null;
+    protected FedoraAuthorizationDelegate fad = null;
 
-    private boolean loggedIn = true;
+    protected boolean loggedIn = true;
```

Comment:
Since the child class, FedoraWebACUserSecurityContext, only needs read-access to `loggedIn`, would it make sense to keep this `private` and create a `protected boolean isLoggedIn()` method?